api/core/tools/provider/builtin/json_process/tools/replace.yaml

```diff
@@ -0,0 +1,91 @@
+identity:
+  name: json_replace
+  author: Mingwei Zhang
+  label:
+    en_US: JSON Replace
+    zh_Hans: JSON 替换
+    pt_BR: JSON Replace
+description:
+  human:
+    en_US: A tool for replacing JSON content
+    zh_Hans: 一个替换 JSON 内容的工具
+    pt_BR: A tool for replacing JSON content
+  llm: A tool for replacing JSON content
+parameters:
+  - name: content
+    type: string
+    required: true
+    label:
+      en_US: JSON content
+      zh_Hans: JSON 内容
+      pt_BR: JSON content
+    human_description:
+      en_US: JSON content
+      zh_Hans: JSON 内容
+      pt_BR: JSON content
+    form: llm
```

Comment:
missing llm_description

Reply:
Thank you for your suggestion. The llm_descriptions have been added.